pkgs/applications/misc/foxtrotgps/default.nix

```
@@ -43,6 +43,11 @@ in stdenv.mkDerivation rec {
   chmod -R u+w $sourceRoot/doc/screenshots
   '';
 
+  # Remove when foxtrotgps supports gpsd 3.23.1
```

Please add more information that makes it easier to determine if this can be removed. Assuming the build currently fails without the change, I suggest

```suggestion
  # Patch for compatibility with gpsd 3.23.1. This was added for foxtrotgps
  # 1.2.2+331. The command can be removed if the build of a newer version
  # succeeds without it.
```